firstvoices/backend/search/signals/dictionary_entry_signals.py

```diff
@@ -0,0 +1,130 @@
+from django.db.models.signals import m2m_changed, post_delete, post_save
+from django.dispatch import receiver
+
+from backend.models.dictionary import (
+    Acknowledgement,
+    DictionaryEntry,
+    DictionaryEntryCategory,
+    Note,
+    Translation,
+)
+from backend.search.tasks.dictionary_entry_tasks import (
+    delete_from_index,
+    update_acknowledgements,
+    update_categories,
+    update_categories_m2m,
+    update_dictionary_entry_index,
+    update_notes,
+    update_translation,
+)
+from firstvoices.celery import link_error_handler
+
+
+# Signal to update the entry in index
+@receiver(post_save, sender=DictionaryEntry)
+def request_update_dictionary_entry_index(sender, instance, **kwargs):
+    if DictionaryEntry.objects.filter(id=instance.id).exists():
+        update_dictionary_entry_index.apply_async(
+            (instance.id,),
+            link_error=link_error_handler.s(),
+            retry=True,
+            retry_policy={
```

Comment:
If these are all the same, it would be worth pulling out a little retry_policy dictionary as a variable